README.md

```diff
@@ -1,19 +1,158 @@
-# Custom Quiz Moodle Plugin
-__User stories__:
-|As a...|I want to...|so that...|
-|------|--------|-----|
-|evaluator|generate quizzes according to clear specifications|I can make sure that every student will have a similar quiz with different questions|
-|evaluator|choose a source for the questions: already loaded questions, new questions from a file or external database|I can choose what set of questions to use for every quiz|
-|evaluator|to have the option to choose from default templates|I can generate the quiz fast, without specifying all the data|
-|evaluator|choose if I want to use new questions or nor|avoid recently used questions|
-|evaluator|keep the default quiz setting available|set the timer, review options, etc.|
-|evaluator|add tags to my questions|specify the difficulty, topic and other relevant data|
-|evaluator|update the last used date of the questions|avoid reusing them|
-|Moodle administrator|avoid question duplicates|create a backup effectively|
-|Moodle administrator|not to crash the server|all the users can use Moodle|
-|student|have the quiz ready when I click "Attempt"|do not waste the allocated time|
-
-
-__References__
- - [QType Template](https://github.com/marcusgreen/moodle-qtype_TEMPLATE/)
- - [Moodle Docs](https://docs.moodle.org/dev/Question_types)
+# Content
+
+0. [Introduction](/README.md#introduction)
+1. [Installation](/README.md#installation)
+2. [Adding Questions](/README.md#adding-questions)
+3. [Managing Questions](/README.md#managing-questions)
+4. [Updating the Local Database](/README.md#updating-the-local-database)
+5. [Exporting Questions](/README.md#exporting-questions)
+6. [References](/README.md#references)
+
+# Introduction
+
+Abbreviations: MXML (Moodle XML)
+
+This plugin is used for creating quizzes that are unique for every student and contains questions with specific difficulties and topics chosen by evaluators.
+The selection of the questions is automated based on options from evaluators.
+The aim is to have a fair distribution of difficulty levels and topics among the unique quizzes created for each student.
+
+In order to use all questions evenly, we keep a timestamp marking when each question was last used.
+There will be a preference to use questions that haven't been recently used.
+As defined in [Managing Questions](README.md#managing-questions), the `last_used` question tag keeps the time of the question was last used, measured in seconds since the Epoch (00:00:00 UTC, January 1, 1970).
```

Comment:
```suggestion
As defined in [Managing Questions](README.md#managing-questions), the `last_used` question tag keeps the time when the question was last used, measured in seconds since the Epoch (00:00:00 UTC, January 1, 1970).
```